sdk/python/packages/flet-ads/src/flet_ads/native_ad.py

```diff
@@ -11,7 +11,7 @@ class NativeAd(BannerAd):
     Renders a native ad.
 
     Raises:
-        AssertionError: When neither [`factory_id`][(c).] nor
+        ValueError: When neither [`factory_id`][(c).] nor
             [`template_style`][(c).] is set.
```

Comment:
Move "Raises" from control description to properties: `factory_id`, `template_style`.